hathor/transaction/storage/transaction_storage.py

```diff
@@ -916,14 +921,18 @@ def get_from_block_height_index(self, height: int) -> bytes:
 
 
 class BaseTransactionStorage(TransactionStorage):
-    def __init__(self, with_index: bool = True, pubsub: Optional[Any] = None) -> None:
+    def __init__(self, with_index: bool = True, with_all_index: bool = False, pubsub: Optional[Any] = None) -> None:
```

Comment:
`with_all_index` is supposed to include the interval-tree indexes, which will not be included when starting a node with only sync-v2 support

Reply:
I've removed all `with_all_index` changes from this PR and the next. Currently I only have them in a backup branch and my stash, since this doesn't add anything I'll leave it to a later PR or drop it.